scipy/optimize/_linprog_doc.py

```diff
@@ -150,6 +151,10 @@ def _linprog_highs_doc(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
 
         Curently, ``None`` always selects ``'steepest-devex'``, but this
         may change as new options become available.
+    mip_rel_gap : double (default: None)
```

Comment:
Open to feedback on this description of the parameter!

Reply:
Please also ask someone for HiGHS to check the description, or take a look at the HiGHS documentation.

Reply:
@jajhall, could I also ask you or someone else from the HiGHS team to weigh in on this option documentation for mip_rel_gap? I couldn't find a good description of the parameter in the HiGHS documentation. Thank you!